### ISSUE TITLE
can only process 64kBit streams

### ISSUE THREAD

**Issue by student1704** [Edzelf/Esp-radio#101]
I have build this project with NODEMCU 0.9 and VS1053 without a TFT. 
But only 64kBit streams are processed properly. With faster streams the tone is unclear.

What to do?

**Comment by Edzelf** [comment]
Please give the URL of a failing stream.  Can you check if this stream works on a Windows machine?

**Comment by student1704** [comment]
I have tried many - alwas the same error. So, it is not a matter of one URL only.
I can hear voices from faster streams but it is not understandable.
Furthermore it seems that volume controll doesn't work or the output signal is extrem stong.
I use the VS1053B. Are the control commands different to the VS1053?

**Comment by Edzelf** [comment]
Please give the URL of a failing stream. Can you check if this stream works on a Windows machine?

**Comment by student1704** [comment]
I use the standard radio.ini and replaced item 6 and 7 with 64kbit streams. Those work any other not.
I think the problem comes from signal XSC. Because when I connect it as proposed - nothing happen. When I tie XCS with 10 K to GND then 64 kbit work. It seems - control commands are not accepted.

**Comment by Edzelf** [comment]
Remove the reset circuit (the 2 diodesand the 1 k resistor).  Then tie the RST pin of the VS1053 to the RST of the NodeMcu or with a pull-up resistor to +3.3V

**Comment by student1704** [comment]
Ok, I made the modifications, but without no success.
I attached 2 screenshots measured with my slow logic analyzer.
![webradio20170725b](https://user-images.githubusercontent.com/30388057/28566472-8e3993b6-712f-11e7-8e0a-d451918e605b.png)
![webradio20170725a](https://user-images.githubusercontent.com/30388057/28566471-8e38ff1e-712f-11e7-9425-6e91c85a86c4.png)

**Comment by Edzelf** [comment]
Yes, XCS seems to be the problem.  It should be low during the clockpulses.

**Comment by KabuloKans** [comment]
Hi Edzelf and congrats for your cool project ! 

Got same issues as student1704 (and the same chip V1053b).  Only streams below 64k are working correctly (without resistor tied to GND),  volume/tones settings aren't working but I can see commands executed in the serial terminal. (Other settings are working on the website version : Next/Prev/Stop/Resume/status/Test)

Got rid of breadboards, removed the reset circuits, etc... 
Nothing worked and XCS always HIGH too.

Any idea ? (Maybe a small difference between V1053 and V1053B ?)

**Comment by KabuloKans** [comment]
Don't really know why but this is working now...
Changed a jack adapter... (don't understand the link between amp and sound distortion...)

Thanks for your work anyway !

**Comment by KabuloKans** [comment]
Is it possible that a capacitor (in ESP) is being in cause ?

While testing I had suddenly no sound from VS1053b output so I thought the problem was amp related.
I checked my cables (and tried a new one) and removed the ESP from my shield (DIY). Unfortunately (and shame on me!) I replaced the ESP upside/down for a second. 
When I replaced the ESP correctly, everything was working.  Maybe I helped a capacitor to discharge.

**Comment by Edzelf** [comment]
In the IDE: Select V1.4 Prebuilt as IwIP Variant. 256 kbps stations will work now.

**Comment by KabuloKans** [comment]
Thank you for helping !
